packages/core/src/constants.ts

```diff
@@ -475,7 +469,7 @@ export type PromptSource = EvaluationV2 | DocumentRunPromptSource
 export type LatitudeToolDefinition = {
   name: LatitudeTool
   internalName: LatitudeToolInternalName
-  definition: (context?: TelemetryContext) => Tool
+  definition: (context?: any) => Tool
```

Comment:
Is this necessary? We still need to import telemetry in core no?

Reply:
better to limit telemetry import as much as possible so that next can tree shake